src/qt/test/rpcnestedtests.cpp

```diff
@@ -127,6 +127,11 @@ void RPCNestedTests::rpcNestedTests()
     RPCConsole::RPCExecuteCommandLine(m_node, result, "rpcNestedTest(   abc   ,   cba )");
     QVERIFY(result == "[\"abc\",\"cba\"]");
 
+// Handle deprecated macro, can be removed once minimum Qt is at least 6.3.0.
+#if (QT_VERSION >= QT_VERSION_CHECK(6, 3, 0))
+#undef QVERIFY_EXCEPTION_THROWN
+#define QVERIFY_EXCEPTION_THROWN(expression, exceptiontype) QVERIFY_THROWS_EXCEPTION(exceptiontype, expression)
```

Comment:
5625840c11db2065a1c8d8de3babb6466eda04d4: note to other reviewers: https://doc.qt.io/qt-6/qtest.html#QVERIFY_THROWS_EXCEPTION